docs/docs/how-tos/connect-via-ssh.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: connect-via-ssh
+title: Connect to Nebari via SSH or SFTP
+description: Set up a remote SSH connection using jupyter-ssh
+---
+
+# Connect to Nebari via SSH or SFTP
+
+Secure Shell Protocol (SSH) is a common protocol for communicating securely over a network connection. Secure File Transfer Protocol (SFTP) is based on SSH and is used to transfer files to and from a remote machine. In this guide, you will learn how to connect to your Nebari deployment via SSH, and how to transfer files using SFTP from your local machine. 
+
+Nebari includes a package called [`jupyterhub-ssh`](https://github.com/yuvipanda/jupyterhub-ssh) in all deployments to allow remote connection to Nebari deployments. On Nebari, SSH connections are set up on port 8022 and SFTP connections on port 8023. 
+
+## Prepare for the connection on the Nebari deployment
+
+Nebari, at its core, is running on a JupyterHub server. You will create a JupyterHub _token_ in order to access your server remotely. To do this, log into your Nebari deployment and navigate to the JupyterHub token page, {your-nebari-domain.org/hub/token}. 
+
+This page is also accessible via the Nebari Home Page:
+
+* If your deployment has `jhub-apps` enabled, you can navigate to this page by selecting the `Token` option under your username in the upper righthand corner. 
+* If your deployment does not have `jhub-apps` enabled, you can click on the `Token` link on the top navigation bar. 
+
+You may add a note and expiration date to the token. You can leave the scopes field blank to give the token all the permissions that your user typically has on JupyterHub (which is the service on which Nebari is running).
+
+## Connect to Nebari via SSH
+
+Before you being, you'll need to have SSH installed on your local machine. From a terminal, use `ssh` to begin the connection. 
+
+`ssh {your-nebari-domain.com} -l {your-username} -p 8022`
```

Comment:
We've typically used angle brackets instead of curly brackets for usage like this. E.g.
![image](https://github.com/user-attachments/assets/77bcb9ad-bbd2-48b3-9373-2dcee94e0d01)

Actually, I seem to see inconsistent usage of angle and curly brackets
![image](https://github.com/user-attachments/assets/a1a435f9-b97a-4f0d-a923-9b8fef72e85b)

Reply:
Yeah, our existing docs are inconsistent. I can switch to the angle brackets instead though. I *think* there might be more of those (after seeing so many of them throw errors in my docusaurus v3 PR...).

Reply:
Could you open an issue to remind us to resolve the curly brackets/angle bracket inconsistency at some point?

Reply:
https://github.com/nebari-dev/nebari-docs/issues/510